src/Pim/Component/Catalog/Builder/ProductBuilder.php

```diff
@@ -184,10 +187,10 @@ public function removeAttributeFromProduct(ProductInterface $product, AttributeI
     /**
      * {@inheritdoc}
      */
-    public function addPriceForCurrency(ProductValueInterface $value, $currency)
```

Comment:
To be done in master, with a `$data = null` in `addPriceForCurrency` (so no BC breack).